bottlecap/src/lifecycle/invocation/context.rs

```diff
@@ -20,13 +22,33 @@ impl Context {
         init_duration_ms: f64,
         start_time: i64,
         enhanced_metric_data: Option<EnhancedMetricData>,
+        tracer_span: Option<Span>,
     ) -> Self {
         Context {
             request_id,
             runtime_duration_ms,
             init_duration_ms,
             start_time,
             enhanced_metric_data,
+            tracer_span,
+        }
+    }
+
+    #[must_use]
+    pub fn from_request_id(request_id: &str) -> Self {
+        Context::new(request_id.to_owned(), 0f64, 0f64, 0, None, None)
+    }
+}
+
+impl Default for Context {
```

Comment:
do we only need this for tests? if so can we wrap it in a test macro? Defaults can cause subtle bugs

Reply:
We do use it for logs processor, where we have default empty at the beginning all the time